web-share/share-null-manual.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share null and undefined values</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Expect null to be converted into the string 'null'. (This isn't good
+        // practice, but it's how Web IDL specifies conversion.) On the other
```

Comment:
Well, especially since that seems like a deliberate choice of the API:

```wedidl
dictionary ShareData {
  USVString title;
  USVString text;
  USVString url;
};
```

If you want null to be handled properly, why not just do so in the IDL? i.e.:

```wedidl
dictionary ShareData {
  USVString? title;
  USVString? text;
  USVString? url;
};
```